Core/src/Surfaces/LineSurface.cpp

```diff
@@ -198,37 +194,39 @@ Acts::BoundToFreeMatrix Acts::LineSurface::boundToFreeJacobian(
   FreeVector freeParams =
       detail::transformBoundToFreeParameters(*this, gctx, boundParams);
   // The global position
-  const Vector3 position = freeParams.segment<3>(eFreePos0);
+  Vector3 position = freeParams.segment<3>(eFreePos0);
   // The direction
-  const Vector3 direction = freeParams.segment<3>(eFreeDir0);
+  Vector3 direction = freeParams.segment<3>(eFreeDir0);
   // Get the sines and cosines directly
-  const double cos_theta = std::cos(boundParams[eBoundTheta]);
-  const double sin_theta = std::sin(boundParams[eBoundTheta]);
-  const double cos_phi = std::cos(boundParams[eBoundPhi]);
-  const double sin_phi = std::sin(boundParams[eBoundPhi]);
+  double cosTheta = std::cos(boundParams[eBoundTheta]);
+  double sinTheta = std::sin(boundParams[eBoundTheta]);
+  double cosPhi = std::cos(boundParams[eBoundPhi]);
+  double sinPhi = std::sin(boundParams[eBoundPhi]);
   // retrieve the reference frame
-  const auto rframe = referenceFrame(gctx, position, direction);
+  auto rframe = referenceFrame(gctx, position, direction);
```

Comment:
The `const`s don't hurt, do they?

Reply:
does not hurt but it is inconsistent IMO. I would either use const on all temporaries or don't const any of them